lib/io/can.cpp

```diff
@@ -9,111 +9,113 @@
 
 namespace hyped::io {
 
-Can::Can(core::ILogger &logger) : logger_(logger)
+Can::Can(core::ILogger &log_) : log_(log_), processors_()
 {
-  processors_ = {};
 }
 
 io::CanResult Can::initialise(const std::string &can_network_interface)
 {
   socket_ = socket(PF_CAN, SOCK_RAW, CAN_RAW);
   if (socket_ < 0) {
-    logger_.log(core::LogLevel::kFatal, "Unable to open CAN socket");
+    log_.log(core::LogLevel::kFatal, "Unable to open CAN socket");
     return io::CanResult::kFailure;
   }
   sockaddr_can socket_address;
   socket_address.can_family  = AF_CAN;
   socket_address.can_ifindex = if_nametoindex(can_network_interface.c_str());
   if (socket_address.can_ifindex == 0) {
-    logger_.log(core::LogLevel::kFatal, "Unable to find CAN1 network interface");
+    log_.log(core::LogLevel::kFatal, "Unable to find CAN1 network interface");
     close(socket_);
     socket_ = -1;
     return io::CanResult::kFailure;
   }
   const int bind_status
     = bind(socket_, reinterpret_cast<sockaddr *>(&socket_address), sizeof(socket_address));
   if (bind_status < 0) {
-    logger_.log(core::LogLevel::kFatal, "Unable to bind CAN socket");
+    log_.log(core::LogLevel::kFatal, "Unable to bind CAN socket");
     close(socket_);
     socket_ = -1;
     return io::CanResult::kFailure;
   }
-  logger_.log(core::LogLevel::kInfo, "CAN socket successfully created");
+  log_.log(core::LogLevel::kInfo, "CAN socket successfully created");
   return io::CanResult::kSuccess;
 }
 
-io::CanResult Can::send(const io::CanFrame &message)
+io::CanResult Can::send(const io::CanFrame &data)
 {
   if (socket_ < 0) {
-    logger_.log(core::LogLevel::kFatal, "Trying to send CAN message but no CAN socket found");
+    log_.log(core::LogLevel::kFatal, "Trying to send CAN data but no CAN socket found");
     return io::CanResult::kFailure;
   }
-  const int num_bytes_written = write(socket_, &message, sizeof(CanFrame));
+  const int num_bytes_written = write(socket_, &data, sizeof(CanFrame));
   if (num_bytes_written != sizeof(CanFrame)) {
-    logger_.log(core::LogLevel::kFatal, "Failed to send CAN message");
+    log_.log(core::LogLevel::kFatal, "Failed to send CAN data");
     return io::CanResult::kFailure;
   }
-  // TODOLater make logger more elegant
-  logger_.log(core::LogLevel::kDebug,
-              "CAN message sent, ID:%i, DATA: %i %i %i %i %i %i %i %i",
-              static_cast<int>(message.can_id),
-              static_cast<int>(message.data[0]),
-              static_cast<int>(message.data[1]),
-              static_cast<int>(message.data[2]),
-              static_cast<int>(message.data[3]),
-              static_cast<int>(message.data[4]),
-              static_cast<int>(message.data[5]),
-              static_cast<int>(message.data[6]),
-              static_cast<int>(message.data[7]));
+  // TODOLater make log_ more elegant
+  log_.log(core::LogLevel::kDebug,
```

Comment:
why did you name this back?

Reply:
log_ is the standard used everywhere else as far as I can see

Reply:
i think the standard has changed to `logger_`. in particular, there was at least one PR for changing `log_` to `logger_` that I've seen